### PR TITLE
Show Filtered to sessions title

### LINKED ISSUE #318: Show Filtered(絞り込み中) to sessions title
## Overview (Required)
- Use "絞り込む"  instead of "絞り込み中", When Filte is working.
Probably The English word is "Filtered". 
image
image

### PR DESCRIPTION
**Pull request by NUmeroAndDev**
## Issue
- close #318 

## Overview (Required)
- Implemented the title when filtered.

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetDaySessionsFragment.kt [hunk 1]
```diff
@@ -97,6 +97,7 @@ class BottomSheetDaySessionsFragment : DaggerFragment() {
                     }
                 }
             groupAdapter.update(items)
+            binding.isFiltered = sessionPagesStore.filtersValue.isFiltered()
```
Comment (takahirom) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetDaySessionsFragment.kt]: Can use add changed callback for sessionPagesStore.filtersValue and do this? 👀

File: model/src/commonMain/kotlin/Filters.kt [hunk 2]
```diff
@@ -23,4 +23,8 @@ data class Filters(
         }
         return roomFilterOk && categoryFilterOk && langFilterOk
     }
+
+    fun isFiltered(): Boolean {
+        return rooms.isNotEmpty() || categories.isNotEmpty() || langs.isNotEmpty()
```
Comment (takahirom) [inline model/src/commonMain/kotlin/Filters.kt]: Nice!

**Comment by takahirom** [comment]
Now, You are number one contributor. 🥇

File: feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetDaySessionsFragment.kt (hunk 1 above)
Comment (NUmeroAndDev) [reply feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetDaySessionsFragment.kt]: @takahirom 
Thanks reviews!

Like this?

https://github.com/DroidKaigi/conference-app-2019/pull/333/commits/a36ca75f5f9e7380e2c63c09b187ac331f95ac8b

**Comment by takahirom** [comment]
Thank you, that is what I thought 👍

**Pull request merged**